uber/site_sections/signups.py

```diff
@@ -69,12 +69,26 @@ def volunteer(self, session, id, csrf_token=None, requested_depts='', message='S
         }
 
     @check_shutdown
-    def shifts(self, session, tgt_date='', state=''):
+    def shifts(self, session, view='', start=''):
         joblist = session.jobs_for_signups()
+        con_days = -(-c.CON_LENGTH // 24) # Equivalent to ceil(c.CON_LENGTH / 24)
```

Comment:
This is all stuff that calculates how many days the "Calendar" and "List" views should show. It shows only the days that a staffer could potentially take shifts for.

If it was just the List view, I'd have it show CON_TOTAL_LENGTH all the time, but that's a bit crowded in Calendar view.